.github/workflows/tests.yaml

```diff
@@ -29,13 +29,16 @@ jobs:
         python -m pip install --upgrade pip
         # get proper PyICU (for natsort)
         sudo apt-get install pkg-config libicu-dev
+        pip install uv
         pip install --no-binary=:pyicu: pyicu
         # install onyo
-        pip install -e ".[tests]"
+        uv venv --python 3.13 onyo-venv
```

Comment:
This is problematic, as it effectively overrides the python version matrix declared above.